### PR TITLE
Add listservers, changegame, and reboot, and fix speedtest, and fix owners + time

### PR DESCRIPTION
**Pull request by dhinakg**
Note: Wiki needs to updated to switch module to speedtest-cli from pyspeedtest.

### CODE REVIEW

**Comment by dhinakg** [comment]
Added speedtest fix.

File: plugins/botutils.py [hunk 1]
```diff
@@ -323,3 +329,27 @@ class perm_admin:
...
+        send = "I am in" 
+        for server in Bot.client.servers:
+            server_names += " {}".format(server.name)
+            send += " {},".format(server.name)
+        send = send[:-1]
+        send = send + "."
+        
+        return message.Message(send)
+
+    if message_in.command == 'changegame':
```
Comment (1byte2bytes) [inline plugins/botutils.py]: Unsecured admin level command

File: plugins/botutils.py [hunk 2]
```diff
@@ -323,3 +329,27 @@ class perm_admin:
 
     if message_in.command == 'ping':
         return message.Message(body='PONG! Bot is up!')
+    
+    if message_in.command == 'restart':
```
Comment (1byte2bytes) [inline plugins/botutils.py]: unsecured admin level command

Comment (dhinakg) [reply plugins/botutils.py]: oh whoops will fix

**Comment by dhinakg** [comment]
Commands fixed.

**Comment by dhinakg** [comment]
Fixed time in certain scenarios.

**Pull request closed without merging**